azurelinuxagent/pa/provision/default.py

```diff
@@ -269,53 +264,12 @@ def deploy_ssh_keypairs(self, ovfenv):
             logger.info("Deploy ssh key pairs.")
             self.osutil.deploy_ssh_keypair(ovfenv.username, keypair)
 
-    def report_event(self, message, is_success=False, duration=0,
-                     operation=WALAEventOperation.Provision):
+    def report_event(self, message, is_success=False, duration=0):
```

Comment:
I would recommend leaving this one change (the ability to specify the operation name) in place. You might not need it now, but you've already written and tested it, and there's zero cost to carrying it along should you need it in the future.